src/constants/colors/index.ts

```diff
@@ -149,17 +152,34 @@ export const guideColors = {
   recommendedCleansersGuide: '#cad1b5',
 };
 
-const colorsCompilation = {
+const sharedColors = {
   ...brandColors,
-  ...legacyColors,
   ...colorAliases,
   ...postcardColors,
   ...guideColors,
 };
 
+const colorsCompilation = {
+  ...sharedColors,
+  ...legacyColors,
+};
+
 const deprecatedProperties = {
   purple80: 'purple80 is deprecated. Use purple85 instead',
   purple60: 'purple60 is deprecated. Use purple70 instead',
 };
 
+// Do not include deprecated properties in PropTypes or it will trigger warning
+// eslint-disable-next-line
+const { purple80, purple60, ...allOtherLegacyColors } = legacyColors;
```

Comment:
I'm open to setting up this spreading and sharing however reviewers think is clearest.

Reply:
I think it's clear the way it's written.